translator/tocwconfig/sampleConfig/emf_and_kubernetes_with_gpu_config.yaml

```diff
@@ -1,1185 +1,1288 @@
 exporters:
```

Comment:
nit - I think this file is really similar to previous mostly formatting changes. Not sure what the procedure for this is but, could we get any similar lines not count as a change to make the pr not as long as it is?